dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -42,7 +42,12 @@ RUN pip2 install 'setuptools==40.8.0'
 # NOTE: Only install enum34 for Python 2, as our version of Python 3 is 3.7, which
 # contains 'enum' as part of the standard library. Installing enum34 there will
 # cause conflicts.
-RUN pip2 install enum34
+RUN pip2 install enum34 \
+        grpcio==1.26.0 \
```

Comment:
We cannot use the default version because upstream is always changing and current version 1.41.0 doesn't build for python2
I have updated to commit to change the grpcio version from 1.26.0 to the 1.39.0 for both  pip2 and pip3.